### PR TITLE
Ui tweak

### LINKED ISSUE #38: Right panel close button styling
Apply similar styling to right panel close button as suggested in #35 

screenshot 2017-08-12 22 07 07

### PR DESCRIPTION
**Pull request by Stephen-Gates**
Fixes #38 #37 

Changes proposed in this pull request:

- update advance qld logo (width of side panels expanded)
- close panel button styling
- app layout improvements
- full menu
- svg icons (need to be resized)
- remove un/freeze header row from context menu 

@mattRedBox

### CODE REVIEW

File: src/renderer/components/Home.vue [hunk 1]
```diff
@@ -40,8 +40,8 @@
           </component>
         </transition>
         <div v-show="sideNavPosition === 'right'" id="sidenav-footer" class="panel-footer">
-          <a href="#" class="left" @click.prevent="sideNavLeft">&lt;</a>
-          <a href="#" class="right" @click.prevent="sideNavRight">&gt;</a>
+          <a href="#" class="left" @click.prevent="sideNavLeft"><span class="btn fa fa-chevron-left fa-2x" /></a>
+          <a href="#" class="right" @click.prevent="sideNavRight"><span class="btn fa fa-chevron-right fa-2x" /></a>
```
Comment (Stephen-Gates) [inline src/renderer/components/Home.vue]: @mattRedBox please look at the panel transitions. Now seem jerky perhaps due to font awesome replacing &lt

**Comment by Stephen-Gates** [comment]
Remaining UI tasks

- add locked.svg/.png to menu items
- in About panel, swap data-curator png out for data-curator-green.svg but need to resize using css
- update app icns using data-curator-green.svg 
- swap fa-icons in menu out for .svgs and resize with css
- swap fa-icons in panels and tabs out for .svgs 
  - add tab
  - close tab
  - close panel
  - next / previous panel
- CSS to prevent ui text from being selectable

File: src/main/menu.js [hunk 2]
```diff
@@ -17,6 +18,7 @@ for (var format in file_formats) {
   }
   open_submenu.push(open_option)
 
+// label should have ... appended to end
```
Comment (Stephen-Gates) [inline src/main/menu.js]: @mattRedBox please append ... to menu item labels

File: src/renderer/components/Home.vue [hunk 3]
```diff
@@ -129,9 +129,11 @@ export default {
...
       toolbarMenus: [{
         name: 'Validate',
-        icon: 'fa-check-circle',
-        sideNavPosition: 'right',
-        sideNavView: 'default'
+        image: '/static/img/validate.svg'
+//        ,
+//        icon: 'fa-check-circle fa-2x',
+//        sideNavPosition: 'right',
+//        sideNavView: 'default'
```
Comment (Stephen-Gates) [inline src/renderer/components/Home.vue]: Validate doesn't have a panel

File: src/renderer/components/Home.vue [hunk 4]
```diff
@@ -141,28 +143,32 @@ export default {
...
       {
         name: 'Export',
         image: '/static/img/export.svg',
-        class: 'down',
-        sideNavPosition: 'right',
-        sideNavView: 'default2'
+        class: 'down'
+//        ,
+//        sideNavPosition: 'right',
+//        sideNavView: 'export'
```
Comment (Stephen-Gates) [inline src/renderer/components/Home.vue]: Export doesn't have a panel. Just ask where to store zip file

File: src/renderer/components/Home.vue [hunk 5]
```diff
@@ -324,18 +330,19 @@ export default {
             {
               items: [
                 {
-                  image: '/static/img/data-curator-120.png'
+                  image: '/static/img/data-curator-128.png',
+//                  image: '/static/img/data-curator-green.svg',
+                  height: '48px'
```
Comment (Stephen-Gates) [inline src/renderer/components/Home.vue]: Prefer to use svg if can be resized

File: src/renderer/components/Home.vue [hunk 6]
```diff
@@ -324,18 +330,19 @@ export default {
...
+                  style: {fontSize: '20px', color: '#0ca831'}
                 },
                 {
-                  label: 'create usable open data',
-                  style: {fontSize: '16px', color: '#0ca831'}
+                  label: 'Create usable open data'
                 },
                 {
-                  label: '1.0.0'
+                  label: '0.1.0'
```
Comment (Stephen-Gates) [inline src/renderer/components/Home.vue]: Need to remember to set this each release

File: src/renderer/components/Home.vue [hunk 7]
```diff
@@ -421,7 +428,7 @@ export default {
       },
       template: sideNavDefaultTemplate
     },
-    default2: {
+    export: {
       data: function() {
```
Comment (Stephen-Gates) [inline src/renderer/components/Home.vue]: Should get rid of this if export panel not required. Same for Validate panel.

File: src/renderer/components/Home.vue [hunk 8]
```diff
@@ -141,28 +143,32 @@ export default {
       },
       {
         name: 'Table',
-        icon: 'fa-table',
+        image: '/static/img/table-properties.svg',
+//        icon: 'fa-table fa-2x',
         sideNavPosition: 'right',
-        sideNavView: 'tablular'
+        sideNavView: 'tablex'
```
Comment (Stephen-Gates) [inline src/renderer/components/Home.vue]: Can't seem to call the panel "Table"

File: src/renderer/components/Home.vue [hunk 9]
```diff
@@ -474,7 +481,7 @@ export default {
       },
       template: sideNavDefaultTemplate
     },
-    tablular: {
+    tablex: {
```
Comment (Stephen-Gates) [inline src/renderer/components/Home.vue]: Can't seem to call the panel "Table"

**Comment by Stephen-Gates** [comment]
@mattRedBox pngs added but code not changed to use them in case you sort out the svg issue

**Comment by mattRedBox** [comment]
Will just merge and fix where needed.

**Pull request merged**